client/packages/common/src/types/schema.ts

```diff
@@ -133,6 +133,13 @@ export enum ActivityLogNodeType {
   InvoiceStatusPicked = 'INVOICE_STATUS_PICKED',
   InvoiceStatusShipped = 'INVOICE_STATUS_SHIPPED',
   InvoiceStatusVerified = 'INVOICE_STATUS_VERIFIED',
+  ItemVariantCreated = 'ITEM_VARIANT_CREATED',
```

Comment:
This looks like stuff from another branch?
Not worried though as it will get merged to develop and this is just the generated branch...

Reply:
Ah yea got generated when I was making my changes